motioneye/utils/__init__.py

```diff
@@ -531,7 +531,8 @@ def parse_editable_mask_file(
     # of the camera image, as it might be different from that of the associated mask;
     # they can be null (e.g. netcams)
 
-    file_name = os.path.join(settings.CONF_PATH, 'mask_%s.pgm' % camera_id)
+    postfix = "_privacy" if mask_class == "privacy" else ""
+    file_name = os.path.join(settings.CONF_PATH, f'mask_{camera_id}{postfix}.pgm')
```

Comment:
Seems like the original implementation (which likely went broken when utils.py was refactored for Python3 support) used also here the `build_mask_file_name` function which is still also being used by `build_editable_mask_file` (line 500). Any reason for not using the same file name builder here also?

Reply:
For reference, here are the changes of the original PR for privacy mask support: https://github.com/motioneye-project/motioneye/pull/1247/files#diff-909205a34627609f47cf1071d96dd8ad03166f0617e4af6390faa7bb50034c61

Reply:
No reason at all. Need to change that.

Reply:
Done